components/card/card.js

```diff
@@ -1 +1,35 @@
-export function createCharacterCard() {}
+export function createCharacterCard() {
+    const characterImageSrc = "https://rickandmortyapi.com/api/character/avatar/1.jpeg";
+    const characterName = "Rick Sanchez";
+    const characterStatus = "Alive";
+    const characterType = null;
+    const characterOccurrences = 51;
```

Comment:
you want to have these as arguments for your function, so you can create Chards for random characters - yours only works for Rick. If you use the function later you want to fill in the different arguments with the content from the fetched data.
```suggestion
export function createCharacterCard(characterImageSrc, characterName, characterStatus, characterType, characterOccurrences) {
```

Reply:
This was hardcoded on purpose! We want to make it dynamic in part2!

Reply:
We would like to keep it as an object which is why we left it hard-coded for this section